packages/teleport/src/DesktopSession/DesktopSession.story.tsx

```diff
@@ -135,3 +140,45 @@ export const ConnectionError = () => (
     connection={{ status: 'failed', statusText: 'some connection error' }}
   />
 );
+export const Performance = () => {
+  const client = fakeClient();
+  var startTime,
+    endTime,
+    i = 0;
+
+  return (
+    <DesktopSession
+      {...props}
+      fetchAttempt={{ status: 'success' }}
+      connection={{ status: 'success' }}
```

Comment:
I think we should rename this `connection` prop to `connectionAttempt` to be more consistent and avoid confusion